include/sparrow/mp_utils.hpp

```diff
@@ -48,6 +35,36 @@ namespace sparrow::mpl
     template <class T, template <class...> class U>
     constexpr bool is_type_instance_of_v = is_type_instance_of<T, U>::value;
 
+    /// A sequence of types, used for meta-programming operations.
+    template <class... T>
+    struct typelist
+    {
+    };
+
+    template <class... Ts, class... Us>
+        requires(!is_type_instance_of_v<Us, typelist> && ...)
+    consteval auto append(typelist<Ts...>, Us...)
```

Comment:
Just add a description/documentation and we're good 👍🏽

Reply:
Added